cap.scad

```diff
@@ -3,12 +3,43 @@ use <threadlib/threadlib.scad>
 // height of the cap (without thread)
 cap_height = 3; // [0:0.1:20]
 // diameter of the cap
-cap_diameter = 52; // [46:0.5:80]
+cap_diameter = 93; // [46:0.5:90]
 // thread turns
 thread_turns = 4; // [1:1:20]
 // thread style
-thread_style = "12-UN-1 3/4";
+thread_style = "G2 3/4";
+//segments
```

Comment:
Not sure if this affects the customizer, but for consistency's sake, let's keep spaces between the `//` and the actual comment.